releases.yml

```diff
@@ -2,18 +2,21 @@ releases:
   4.13.5:
     assembly:
       basis:
-        brew_event: 52323250
+        brew_event: 52501133
         reference_releases:
-          x86_64: 4.13.0-0.nightly-2023-07-05-213721
+          s390x: 4.13.0-0.nightly-s390x-2023-07-14-070246
+          ppc64le: 4.13.0-0.nightly-ppc64le-2023-07-14-070247
+          aarch64: 4.13.0-0.nightly-arm64-2023-07-14-070248
+          x86_64: 4.13.0-0.nightly-2023-07-14-070245
       group:
         advisories:
-          extras: 116849
-          image: 116848
-          metadata: 116850
-          rpm: 116847
-          microshift: 116851
-        release_jira: ART-7221
-        upgrades: 4.12.16,4.12.17,4.12.18,4.12.19,4.12.20,4.12.21,4.12.22,4.12.23,4.12.24,4.13.0,4.13.1,4.13.2,4.13.3,4.13.4
+          extras: -1
+          image: -1
+          metadata: -1
+          rpm: -1
+          microshift: -1
+        release_jira: ART-0
+        upgrades: 4.12.16,4.12.17,4.12.18,4.12.19,4.12.20,4.12.21,4.12.22,4.12.23,4.12.24,4.12.25,4.13.0,4.13.1,4.13.2,4.13.3,4.13.4
       issues:
```

Comment:
```suggestion
```

Reply:
Let's remove the issues section, since these are all fips bugs, that we're not shipping